### DISCUSSION TITLE
Chatfunktion

### DISCUSSION THREAD

**Discussion by CitaroMH** [Traewelling/traewelling#2031] [Ideas / Feature Requests]
Hallo zusammen,

wäre es möglich, vielleicht noch eine Chatfunktion oder Private Nachrichten einzuführen?

So könnte man nämlich auch neue Leute kennen lernen und vielleicht mal Reisegemeinschaften bilden.

Viele Grüße und einen schönen Samstag

Moritz

**Comment by DhaDenny** [comment]
Das wäre in der Tat eine interessante Funktion für Leute, die im selben Zug fahren. So kann man ja auch Kontakte knüpfen. Ich sehe jetzt so keine Nachteile in dieser Funktion und sonderlich aufwendig sollte das wahrscheinlich nicht sein.

**Comment by MrKrisKrisu** [comment]
Die Discussion hier war zwar früher da, aber ich mache die hier mal zu und verweise auf #3033, weil da schon mehr geschrieben wurde.
